logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java

```diff
@@ -5,10 +5,13 @@
 import java.util.function.Supplier;
 
 import kr.pe.kwonnam.slf4jlambda.LambdaLogger;
+import org.checkerframework.checker.nullness.qual.Nullable;
 
 public class PrefixedLambdaLogger implements IPrefixedLambdaLogger {
   private final LambdaLogger logger;
   private final String className;
+  private final ThreadLocal<@Nullable String> bufferedTimerMessage;
```

Comment:
Tip: `ThreadLocal<T>` is easiest to conceptualize as `ConcurrentHashMap<Long /* thread id */, T>`